extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -85,8 +86,10 @@ class AmpInstagram extends AMP.BaseElement {
     const image = this.win.document.createElement('amp-img');
     // This will redirect to the image URL. By experimentation this is
     // always the same URL that is actually used inside of the embed.
-    image.setAttribute('src', 'https://www.instagram.com/p/' +
-        encodeURIComponent(this.shortcode_) + '/media/?size=l');
+    const imgSrc = resolveUrlAttr('amp-img', 'src',
```

Comment:
In my opinion, we shouldn't proxy an Instagram image. We should either find a way to get it from their cookieless CDN or exclude it from prerendering. The reasons are:

1. They will most likely immediately throttle any AMP CDN
2. This prefetch will miss the cache when the actual widget is downloaded, which makes pre-fetch a lot less useful.